Blockzilla/URLBar.swift

```diff
@@ -713,11 +714,27 @@ extension URLBar: AutocompleteTextFieldDelegate {
     }
 
     func autocompleteTextFieldShouldReturn(_ autocompleteTextField: AutocompleteTextField) -> Bool {
+        // Checks if the user used the autocomplete to finish their query
+        let domains = domainCompletion.getDomains()
```

Comment:
Ideally telemetry is no-cost. I am not familiar with this code, getDomains() may have a cost I am not aware of. Is this guaranteed cheap to call?